src/main/java/dmytro/hadiuchko/springboot/controller/BookController.java

```diff
@@ -18,41 +21,50 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RestController
 @RequestMapping(value = "/books")
 @RequiredArgsConstructor
 public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all products", description = "Get list with sorted products")
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
     }
 
     @GetMapping("/{id}")
+    @Operation(summary = "get product by id", description = "return book by id")

```

Comment:
```suggestion
    @Operation(summary = "Get product by id", description = "Return book by id")
```
let's make those pretty